charts/kubernetes-agent/templates/tentacle-deployment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
             - name: "ACCEPT_EULA"
               value: {{ .Values.agent.acceptEula | quote }}
             - name: "TargetName"
-              value: {{ .Values.agent.targetName | quote }}
+              value: {{ .Values.agent.name | quote }}
             - name: "ServerCommsAddresses"
```

Comment:
I was contemplating whether or not to put ServerCommsAddresses under its own category namespace, and everything else under another. Something like `agent.ServerCommsAddresses` and everything else under `agent.initial._...`
The idea is to provide some distinction between "only-on-install" options that can't be updated in Tentacle after the first time install vs updatable fields. However, the extra namespace feels a bit overkill and can add confusion.

Maybe it's best to simply be clear with a comment or two in the docs.